notices.json

```diff
@@ -1,4 +1,41 @@
 [
+      {
+    "id": "1",
+    "conditions": {
+      "audience": "sysadmin",
+      "clientType": "all",
+      "serverVersion": ["<6.00"],
+      "displayDate": "> 2021-10-13T00:00:00Z <= 2021-11-01T00:00:00Z",
+      "repeatable": "false" 
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Mattermost 6.0 is here!",
+        "description": "Mattermost has expanded beyond channel-based messaging and offers Playbooks and Boards as core features to our open source developer collaboration platform!",
+        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/6.0.png",
+        "actionText": "Learn more",
+        "actionParam": "https://mattermost.com/blog/mattermost-v6-00/"
+      }
+    }
+  }
+ {
+    "id": "2",
```

Comment:
The challenge with this logic is that the notice will appear immediately when a new admin is onboarding (ie it's the first thing admins will see after being dropped in the product during onboarding). 

If these type of notices are needed then we probably also need to add a condition that will filter based on account age (ie only show if the account age is > 24 hours). This logic doesn't exist yet so will need dev investment.

One option in the mean time would be adding a condition of posts AND users. So for example, if posts > 3 AND users = 1, then fire this notice. It's not perfect, since we can't guarantee that an admin will send messages, but it at least ensures that we aren't disrupting the onboarding flow. cc// @jasonblais @wiersgallak thoughts?

If we want to go ahead with the notice with that tweak I've opened a PR against the `release` branch here: https://github.com/mattermost/notices/pull/222

Reply:
I think that is a good compromise.  Ideally, we'd have the age of server, but we want to implement sooner if we can.  I think we can do > 5 posts to give it a bit more time.